### PR TITLE
feature: added result

### PR DESCRIPTION
**Pull request by omgaikwad**
feature: added result
Preview: https://deploy-preview-3--gallant-pasteur-062544.netlify.app/pages/result/result.html

### CODE REVIEW

**Review by fahd23 — APPROVED** [review]
The code looks good to me

File: pages/question/question.css [hunk 1]
```diff
@@ -24,6 +24,7 @@
   justify-content: space-around;
   gap: 2rem;
   margin-top: 4rem;
+  margin-bottom: 4rem;
```
Comment (RakshithBhat03) [inline pages/question/question.css]: Just a small suggestion, maybe you can use the shorthand property for margin or you can use `margin-block` property for setting margins on the y-axis.

File: pages/result/result.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        <!-- Question -->
+        <h3 id="question">What is Floki's profession?</h3>
+        <div class="option-container">
+          <div class="option correct-option">Ship Builder</div>
+          <div class="option wrong-option">Carpenter</div>
+          <div class="option">Blacksmith</div>
+        </div>
+        <!-- Question -->
+        <h3 id="question">
```
Comment (RakshithBhat03) [inline pages/result/result.html]: Use different ids or use a common class.

File: pages/result/result.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            To attend a rite because he and Lagertha are unable to conceive a
+            child.
+          </div>
+          <div class="option">To meet with the seer.</div>
+          <div class="option">
+            To gain King Horik's blessing for another raid on England.
+          </div>
+        </div>
+        <!-- Question -->
+        <h3 id="question">
```
Comment (RakshithBhat03) [inline pages/result/result.html]: Same as suggested above

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested some changes. Apart from that, the code looks clean and well structured.

File: pages/question/question.css (hunk 1 above)
Comment (omgaikwad) [reply pages/question/question.css]: Sure

File: pages/result/result.html (hunk 2 above)
Comment (omgaikwad) [reply pages/result/result.html]: Thanks for pointing it out, I will change it to class.

File: pages/result/result.html (hunk 3 above)
Comment (omgaikwad) [reply pages/result/result.html]: Sure!

**Pull request merged**